site.html

```diff
@@ -0,0 +1,123 @@
+
+<!DOCTYPE html>
+<html lang="pt-br">
+	<head>
+		<meta charset="UTF-8">
+		<title>WOMAKERS CODE</title>
+		<link rel="icon" href="images/icon.ico" />
+	</head>
+	<body>
+		<header>
+			<nav>
+				<ul>
+					<li><a href=""><img src="images/logo.png" width= "160" height= "50"/></a></li>
+					<li><a href="">HOME</a></li>
+					<li><a href="">SERVIÇOS</a></li>
+					<li><a href="">PORTFOLIO</a></li>
+					<li><a href="">SOBRE MIM</a></li>
+					<li><a href="">CONTATO</a></li>
+				</ul>
+			</nav>
+		</header>
+		<section>
+			<figure>
+				<img src="images/Slider.png" alt="Image-Slider" width="100%" height="100%"/>
+				<figcaption>
+					<h1>BEM VIND@</h1>
+					<h2>CURABITUR IACULIS</h2>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc faucibus tellus ipsum, sed sollicitudin metus ornare nec. Sed nulla sapien.</p>
+				</figcaption>
+			</figure>
+			<figure>
+				<img src="images/slider01.png" height= "250px" alt="Image-Slider01" />
+			</figure>
+			<figure>
+				<img src="images/Slider02.png" height= "250px" alt="Image-Slider02" />
+			</figure>
+			<figure>
+				<img src="images/Slider03.png" height= "250px" alt="Image-Slider03" />
+			</figure>
+		</section>
+		<section>
+			<h1>Serviços</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec euismod diam eget ex rhoncus, sed.</p>
+			<hr />
+			<article>
+				<img src="images/Service1.png" alt="Service1" />
+				<h3>Websites</h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque.</p>
+			</article>
+			<article>
+				<img src="images/Service2.png" alt="Service2" />
+				<h3>Templates</h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque.</p>
+			</article>
+			<article>
+				<img src="images/Service3.png" alt="Service3" />
+				<h3>Temas WordPress</h3>
+				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque.</p>
+			</article>
+		</section>
+		<section>
+			<h1>Meus Trabalhos</h1>
+			<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed varius consectetur felis ut condimentum. Aliquam.</p>
+			<hr />
+			<nav>
+				<ul>
+					<li><a href="">Tudo</a></li>
+					<li><a href="">Website</a></li>
+					<li><a href="">Template</a></li>
+					<li><a href="">Tema</a></li>
+				</ul>
+			</nav>
+			<article>
+				<img src="images/Portfolio01.png" height= "250px" alt="Portolio01" />
```

Comment:
Não é necessário especificar a medida px (pixel) nos atributos do elemento `<img>`

```suggestion
				<img src="images/Portfolio01.png" height= "250" alt="Portolio01" />
```